README.md

```diff
@@ -85,7 +85,8 @@ req['Authorization'] = JWTSignedRequest.sign(
   path: req.path,
   headers: {"Content-Type" => "application/json"},
   body: "",
-  key_id: 'my-key-id',
+  key_id: 'my-key-id',                    # used for looking up key and kid header
+  target_key_id: 'my-alt-key-id',         # optionally override kid header value
```

Comment:
I haven't used the `KeyStore` yet so I may be missing enough context to comment on this PR. With that in mind, this feels backwards to how I would have thought about it. The `key_id` to me feels like it refers to the JWT `kid`. Would a `lookup_key_id` make sense and still solve the problem?

Reply:
Yes, so long as target or lookup can be overridden then problem solved. 

👍 Great suggestion: `key_id` is more akin to `kid` so it makes more sense to override the lookup.